test/resources/V1_1/eurostag-tutorial-example1-with-terminalMock-ext.xml

```diff
@@ -1,44 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<iidm:network xmlns:iidm="http://www.powsybl.org/schema/iidm/1_1" xmlns:mock="http://www.powsybl.org/schema/iidm/ext/terminal_mock/1_1" id="sim1" caseDate="2013-01-15T18:45:00.000+01:00" forecastDistance="0" sourceFormat="test">
+<iidm:network xmlns:iidm="http://www.powsybl.org/schema/iidm/1_1" xmlns:mock="http://www.powsybl.org/schema/iidm/ext/terminal_mock/1_1" id="sim1" caseDate="2013-01-15T18:45:00+01:00" forecastDistance="0" sourceFormat="test">
```

Comment:
Not sure to understand why this file has to be modified.

Reply:
This file (V1_1/eurostag-tutorial-example1-with-terminalMock-ext.xml) was erroneous but was not read as the concerning test was executed for every previous versions (i.e only for V1.0) : 
```
BOOST_AUTO_TEST_CASE(TerminalExtension) {
    ....
    test::converter::RoundTrip::roundTripAllPreviousVersionedXmlTest("eurostag-tutorial-example1-with-terminalMock-ext.xml");
}
```
=> updated

Reply:
In the Java code, this file is used in all versions:
```
    @Test
    public void testTerminalExtension() throws IOException {
        Network network2 = roundTripXmlTest(EurostagTutorialExample1Factory.createWithTerminalMockExt(),
                NetworkXml::writeAndValidate,
                NetworkXml::read,
                getVersionedNetworkPath("eurostag-tutorial-example1-with-terminalMock-ext.xml", CURRENT_IIDM_XML_VERSION));
        Load loadXml = network2.getLoad("LOAD");
        TerminalMockExt terminalMockExtXml = loadXml.getExtension(TerminalMockExt.class);
        assertNotNull(terminalMockExtXml);
        assertSame(loadXml.getTerminal(), terminalMockExtXml.getTerminal());

        // backward compatibility
        roundTripAllPreviousVersionedXmlTest("eurostag-tutorial-example1-with-terminalMock-ext.xml");
    }

```